### ISSUE TITLE
Pumping should have a start time and be creatable from a timer

### ISSUE THREAD

**Issue by hsharrison** [babybuddy/babybuddy#573]
Just setting this up and looks like it meets pretty much all our needs (for the last child we used an Android app which was good but didn't sync between us), this is one thing suggested by my wife. Let's see if I can find time to look into it myself, but thought I may as well open an issue.

**Comment by cdubz** [comment]
Makes sense! Will watch out for a PR. Shouldn’t be too complicated.

**Comment by tzw9373** [comment]
I don't have the skills to assist with a PR, but just wanted to say this feature was requested by my wife as well and would be greatly appreciated!

**Comment by moejay** [comment]
I'll give this a shot, my wife loves the app, and requested the exact same feature.

**Comment by johnmerlie** [comment]
I've begun to look at this, as my wife had the same request. I've started working on this at [johnmerlie:babybuddy](https://github.com/johnmerlie/babybuddy/tree/pumping-end-time). 

I think i've got most of the features down, as it was largely a process of mimicking the setup of the `feeding` entity.

I've tested in-situ (via `gulp`) by adding pumping entries via the list view as well as the quick timer - but as this is my first experience with django migrations I'm not sure if i've completely addressed that from the standpoint of modifying the existing schema. The strategy I'm thinking is to simply convert the `pumping.time` field to a `pumping.start` and add a `pumping.end` field - populating the existing `pumping.start` time in that field to ensure duration can be calculated but ultimately showing the record as a 0 duration pumping entry. 

I'll also still need to look at the tests, as I know there will be some additional work needed there.
